### ISSUE TITLE
Accessible labels for column header filter text boxes

### ISSUE THREAD

**Issue by lordofthemoon** [olifolkerd/tabulator#4673]
**Is your feature request related to a problem? Please describe.**

UK public sector requires websites and applications to comply with WCAG 2.1 AA or greater.  Section 2.4.6 of the WCAG spec discusses headings and labels.  MDN says that this [can be met](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#form_elements_must_be_labeled) by ensuring that all form elements are labelled.

When column header filters exist for a Tabulator table, there is no label for the text boxes.

**Describe the solution you'd like**

Use the title of the column as the label for the header filter text box.  The `<input>` elements would need IDs that the title `<label>`s can reference

**Describe alternatives you've considered**

None

**Additional context**

WCAG guidelines:
https://www.w3.org/TR/WCAG21/#headings-and-labels
https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#form_elements_must_be_labeled

**Comment by pgmccann** [comment]
I would suggest that this isn't a suggested feature, but rather an enhancement or possibly even a bug.

**Comment by mary-wk** [comment]
Without accessible labels for column header filters, any table fails  Level A compliance tests.
